store.css

```
@@ -0,0 +1,157 @@
+:root {
+  --orange-dark: #de6e49;
+}
+
+/*body*/
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/*header*/
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 5rem;
+  background: rgba(196, 196, 196, 0.1);
+}
+
+.logo {
+  width: 2rem;
+  height: 2rem;
+}
+/*navbar*/
+nav {
+  display: flex;
+}
+
+ul {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  font-size: 20px;
+  margin: 5px;
+  padding: 5px;
+}
+li {
+  align-self: center;
+  padding: 1rem;
+}
+
+a {
+  text-decoration: none;
+  color: grey;
+}
+a:hover {
+  color: orange;
+}
+/*main*/
+main {
+  display: flex;
+}
+
+.form-container {
+  margin: 7%;
+  flex-direction: column;
+  font-style: normal;
+}
+
+.karma {
+  color: var(--orange-dark);
+  font-size: 3rem;
+  font-weight: bold;
+  margin-top:5px;
+}
+
+.fieldset {
```

Here you want to target the fieldset tag rather than a class, so you should remove the . before the name:
```suggestion
fieldset {
```
more info on CSS targeting here: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors